payments-model/src/main/java/com/stripe/android/repository/ConsumersApiService.kt

```diff
@@ -45,6 +45,7 @@ interface ConsumersApiService {
     suspend fun lookupConsumerSession(
         email: String,
         requestSurface: String,
+        doNotLogConsumerFunnelEvent: Boolean = false,
```

Comment:
Nit: Should we remove the default param here, so that callers are forced to think about which values to provide?